trl/trainer/grpo_trainer.py

```diff
@@ -1824,10 +1824,9 @@ def _compute_loss(self, model, inputs):
 
         # From here, log_importance_weights (and all subsequent tensors, coef_1, coef_2, etc.) shape depends on
         # importance_sampling_level: "token" level: (B, T); "sequence" level: (B, 1)
-        if self.loss_type in ["grpo", "bnpo", "dr_grpo", "dapo"]:
+        if self.loss_type == "cispo":
             clamped_ratios = torch.clamp(coef_1, max=self.epsilon_high).detach()
             per_token_loss = -clamped_ratios * advantages.unsqueeze(1) * per_token_logps
-
         else:
```

Comment:
to be a tiny bit more consistent
```suggestion
        elif self.loss_type in ["grpo", "bnpo", "dr_grpo", "dapo"]
```